x/tradeshield/client/cli/tx_spot_order.go

```diff
@@ -156,3 +156,29 @@ func readPositionRequestJSON(filename string) ([]uint64, error) {
 
 	return positions, nil
 }
+
+func CmdCancelAllSpotOrders() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "cancel-all-spot-orders",
+		Short:   "Cancel all spot-orders",
+		Example: "elysd tx tradeshield cancel-all-spot-orders --from=bob --yes --gas=1000000",
+		Args:    cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgCancelAllSpotOrders(clientCtx.GetFromAddress().String())
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
```

Comment:
i don't think we need this file, autocli can handle it, just have to define in autocli. could you please check ? same for above cli file

Reply:
I think we are using custom CLI for tx till v0.51